src/main/java/com/zendesk/maxwell/producer/MaxwellKafkaProducer.java

```diff
@@ -110,9 +111,21 @@ public MaxwellKafkaProducer(MaxwellContext context, Properties kafkaProperties,
 		this.inflightMessages = new InflightMessageList();
 	}
 
+	private Integer getNumPartitions(String topic) {
+		try {
+			return this.kafka.partitionsFor(topic).size(); //returns 1 for new topics
+		} catch(TimeoutException e) {
+			LOGGER.error("Topic '" + topic + "' name does not exist. Exception: " + e.getLocalizedMessage());
+			throw e;
```

Comment:
yeah, saw that.  I also noted in a separate thread that even a `listTopics` doesn't really properly deal with the subject of auto-creation.  Regardless, it doesn't look like there's an easy win so maybe we hack around it?  I'd say we should lower the default metadata timeout to like 3 seconds so that it doesn't take us 60 seconds to crash, and then just keep the crash-on-TimeoutException lameness for now.